columnphysics/icepack_therm_vertical.F90

```diff
@@ -2731,10 +2731,21 @@ subroutine icepack_step_therm1(dt,                        &
                                  prescribed_ice=prescribed_ice)
 
             if (icepack_warnings_aborted(subname)) then
-               write(warnstr,*) subname, ' ice: Vertical thermo error, cat ', n
-               call icepack_warnings_add(warnstr)
-               return
+               
+               if ((vicen(n)/aicen(n)) > real(5e-2)) then
```

Comment:
@SmithJos13 I am not sure how you end up `5e-2` ratio but do you think tat it is applicable for all the cases. I also wonder if this change is really needed in this level. At this point, the main aim is to keep the changes minimal and isolated. So, any other configuration needs to work as it is but your changes will allow to run your case.